src/common/src/buffer/bitmap.rs

```diff
@@ -631,18 +637,28 @@ unsafe impl TrustedLen for BitmapIter<'_> {}
 
 pub struct BitmapOnesIter<'a> {
     bitmap: &'a Bitmap,
-    idx: usize,
+    cur_idx: usize,
+    cur_bits: Option<usize>,
```

Comment:
I suspect we can save a usize here.

```rs
size_of Option<usize>: 16
// change to usize: 8
```

None case is terminal. Hence `cur_idx` is no longer needed when that happens.
We can encode `None` case as `cur_idx == usize::MAX`, as I don't think we will have bitmap of such a large size.
Then `cur_bits` will just be `usize` and we use `cur_idx` as the discriminant for `None` case.

Don't need to implement in this PR as it adds complexity, or possibly ever, just noting it for my own reference.
Only if Heap profile shows bitmap iter takes an unreasonable amount of memory, we should optimize it (not a priority now).